recipes-bsp/u-boot/files/0013-iot2050-config-the-setexpr-cmd.patch

```diff
@@ -0,0 +1,27 @@
+From 3fe2e1c82a026f5282bcddcd49087627c87e6956 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Tue, 8 Mar 2022 13:42:34 +0800
+Subject: [PATCH 13/14] iot2050: config the setexpr cmd
+
+enable the setexpr cmd under u-boot.
```

Comment:
No reason provided.

Reply:
> 
this one is needed, when load the dtbo, it needs the support of this command